doc/user-guides/authenticated-rl-for-app-developers.md

```diff
@@ -101,6 +101,15 @@ curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toy -i
 # HTTP/1.1 200 OK
 ```
 
+> **Note**: If the command above fails to hit the Toy Store API on your environment, try forwarding requests to the service and accessing over localhost:
+>
+> ```sh
+> kubectl port-forward -n istio-system service/istio-ingressgateway 9080:80 2>&1 >/dev/null &
+> export GATEWAY_URL=localhost:9080
+> curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toys -i
```

Comment:
nitpick: I like this one more
```
curl --resolve api.toystore.com:9080:127.0.0.1 -v "http://api.toystore.com:9080/toys -i
```

Reply:
It was not about the GATEWAY_URL. This one also works for me

```
curl --resolve api.toystore.com:$INGRESS_PORT:$INGRESS_HOST -v "http://api.toystore.com:$INGRESS_PORT/toys -i
```

Reply:
I'm sorry I don't think I follow, if we're removing the use of the `GATEWAY_URL` var in this one curl command doesn't that obfuscate the usage for the others. Shouldn't we make them all match?

Reply:
it's a nitpick. I just wanted to point out that using `--resolve` flag of `curl` is better than setting the Host header. If it makes more sense as you say, go with it.